### PR TITLE
LPS-168445 Using the same locale as the page being rendered

### PR DESCRIPTION
**Pull request by 4lejandrito**
Fixes [LPS-168445](https://issues.liferay.com/browse/LPS-168445). 

With these changes we pass the locale from the original request to the headless APIs in the `Accept-Language` header.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/servlet/RESTClientHttpRequest.java [hunk 1]
```diff
@@ -51,6 +59,11 @@ public Enumeration<String> getHeaders(String name) {
 				Arrays.asList(ContentTypes.APPLICATION_JSON));
 		}
 
+		if (StringUtil.equalsIgnoreCase(name, "Accept-Language")) {
+			return Collections.enumeration(
+				Arrays.asList(_locale.toLanguageTag()));
```
Comment (ccorreagg) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/servlet/RESTClientHttpRequest.java:64]: I think Collections.enumeration is not necessary as a List already implements a Collection interface

**Review by ccorreagg — COMMENTED** [review]
Great!

**Comment by 4lejandrito** [comment]
ci:forward

**Review by ccorreagg — APPROVED** [review]

**Pull request closed without merging**